src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -200,4 +162,67 @@ private Entity findTerm(
     List<Entity> queryList = datastore.prepare(query).asList(FetchOptions.Builder.withDefaults());
     return queryList;
   }
+
+  private double getToxicityScore(String comment) throws IOException {
+    HttpURLConnection commentAnalyzerConnection = creatingPostRequestCommentAnalyzer();
+    // Create JSON request.
+    JSONObject jsonObjectCommentAnalyzer = jsonObjectCommentAnalyzerRequest(comment);
```

Comment:
This variable name is misleading. This is a response from an analysis request, not an analyzer.